tests/integration/polygonParts/polygonParts.spec.ts

```diff
@@ -16,12 +16,24 @@ import { SERVICES } from '../../../src/common/constants';
 import type { DbConfig } from '../../../src/common/interfaces';
 import { Part } from '../../../src/polygonParts/DAL/part';
 import { PolygonPart } from '../../../src/polygonParts/DAL/polygonPart';
+import {
+  createInitPayloadRequest,
+  franceFootprint,
+  germanyFootprint,
+  intersectionWithItalyFootprint,
+  intersectionWithItalyRequest,
+  italyFootprint,
+  italyWithoutIntersection,
+  separatePolygonsRequest,
+  worldFootprint,
+  worldMinusSeparateCountries,
+} from '../../mocks/requestsMocks';
 import type { PolygonPartsPayload } from '../../../src/polygonParts/models/interfaces';
 import polygonEarth from './data/polygonEarth.json';
 import polygonHole from './data/polygonHole.json';
 import polygonHoleSplitter from './data/polygonHoleSplitter.json';
 import { INITIAL_DB } from './helpers/constants';
-import { HelperDB, createDB, createPolygonPartsPayload } from './helpers/db';
+import { HelperDB, createDB, createPolygonPartsPayload, deleteDB } from './helpers/db';
```

Comment:
Remove unused

Reply:
see comment below about the usage of the deleteDB function

Reply:
Noticed, please add comment about the future use